site/tests/app/libraries/GradingQueueTester.php

```diff
@@ -16,6 +16,7 @@
 class GradingQueueTester extends BaseUnitTest {
```

Comment:
either expand the existing tests or add some new ones to verify the changed behavior in `GradingQueue` model.